src/Operators/remapping.jl

```diff
@@ -290,3 +290,103 @@ function local_weights(space::AbstractSpace)
     wj = space.local_geometry.WJ
     return vec(parent(wj))
 end
+
+## RegularLatLong interpolation
+struct RLLRemap{
+    T <: Meshes.RegularLatLongMesh,
+    S <: AbstractSpace,
+    M <: AbstractMatrix,
+    E <: AbstractVector,
+    P <: AbstractVector,
+}
+    target::T # rll mesh
+    source::S # source space
+    map::M # linear mapping operator
+    elems::E # elements associated with target pts
+    p::P # coordinate permutation vector
+end
+
+function RLLRemap(target_mesh::Meshes.RegularLatLongMesh, source_space)
+
+    source_mesh = Spaces.topology(source_space).mesh
+    latitude, longitude = target_mesh.lat, target_mesh.long
+
+    # store latlong coords and associated elements
+    coords = []
+    for lat in latitude
+        for long in longitude
+            coord = Geometry.LatLongPoint(lat, long)
+            elem = Meshes.containing_element(source_mesh, coord)
+            push!(coords, (coord = coord, elem = elem))
+        end
+    end
```

Comment:
may be more efficient to do this as a comprehension:
```suggestion
    coords = [begin
            coord = Geometry.LatLongPoint(lat, long)
            elem = Meshes.containing_element(source_mesh, coord)
            (coord = coord, elem = elem)
        end for long in longitude, lat in latitude]
```